ui_framework/doc_site/src/views/table/controlled_table.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiToolBarSection,
+  KuiButton,
+  KuiButtonIcon,
+  KuiTable,
+  KuiToolBarText,
+  KuiControlledTable,
+  KuiTableRowCellLiner,
+  KuiPager,
+  KuiTableRow,
+  KuiTableRowCell,
+  KuiTableHeaderCell,
+  KuiToolBarFooterSection,
+  KuiToolBarFooter,
+} from '../../../../components';
+
+export class ControlledTable extends React.Component {
+  getSampleRowData() {
+    return [
+      [
+        <a className="kuiLink" href="#">Alligator</a>,
+        <div className="kuiIcon kuiIcon--success fa-check"></div>,
+        'Tue Dec 06 2016 12:56:15 GMT-0800 (PST)',
+        '1'
+      ],
+      [
+        <a className="kuiLink" href="#">Boomerang</a>,
+        <div className="kuiIcon kuiIcon--success fa-check"></div>,
+        'Tue Dec 06 2016 12:56:15 GMT-0800 (PST)',
+        '10'
+      ],
+      [
+        <a className="kuiLink" href="#">Celebration</a>,
+        <div className="kuiIcon kuiIcon--warning fa-bolt"></div>,
+        'Tue Dec 06 2016 12:56:15 GMT-0800 (PST)',
+        '100'
+      ],
+      [
+        <a className="kuiLink" href="#">Dog</a>,
+        <div className="kuiIcon kuiIcon--error fa-warning"></div>,
+        'Tue Dec 06 2016 12:56:15 GMT-0800 (PST)',
+        '1000'
+      ],
+    ];
+  }
+
+  getPager() {
+    return (
+      <KuiPager
+        startNumber={ 1 }
+        hasNextPage={ true }
+        hasPreviousPage={ false }
+        endNumber={ 10 }
+        totalItems={ 100 }
+        onNextPage={ () => {} }
+        onPreviousPage={ () => {} }
+      />
+    );
+  }
+
+  getTableRows() {
```

Comment:
What do you think of using `renderTableRows`, `renderPager`, etc? I know we've used `get` in some other places, but I think I'm used to seeing `get` and `set` used mostly with getters and setters. `render` seems more appropriate to me because it indicates a clear connection to the `render()` method.